.rubocop.yml

```diff
@@ -2,4 +2,6 @@
 #   decidim: .rubocop.yml
 
 Layout/SpaceAroundOperators:
-  Enabled: false
\ No newline at end of file
+  Enabled: false
+Metrics/LineLength:
+  Max: 120
```

Comment:
Why you are changing the max LineLength? Is this the same than Decidim:Decidim?

Reply:
I guess that the standard is 80. Many lines are between 80 and 120 characters so I decided to change it to not have to "cut" lines so many times.